roles/android-sdk-packages/tasks/main.yml

```
@@ -65,7 +65,8 @@
     owner: "{{ param_user }}"
     group: staff
 
-- name: chmod a+x to android-sdk
+- name: chmod 755 to android-sdk
   become: yes
-  shell: "(chmod -R a+w /usr/local/Caskroom/android-sdk || chmod -R a+W /Users/vagrant/Library/Developer/Xamarin/android-sdk-macosx)"
+  shell: "(chmod -R 755 /usr/local/Caskroom/android-sdk || chmod -R 755 /Users/vagrant/Library/Developer/Xamarin/android-sdk-macosx)"
+  ignore_errors: yes
```

what's the logic for ignoring errors?

---

on vs4mac stacks the first path is not there and ansible throws an error on it

---

OK. Can you use the file module then?
https://docs.ansible.com/ansible/latest/modules/file_module.html

two sections like this: 
```
- name: Recursively change ownership of a directory
  file:
    path: /usr/local/Caskroom/android-sdk
    state: directory
    recurse: yes
    mode 0755
    owner: ??
    group: ??
```

If the dir doesn't exist, it will make it, if it does, it will traverse it. No harm in having the empty dir there. But it is better to use the module than use a flakey shell command